deployments/ansible/roles/collector/templates/collector-service-proxy.conf.j2

```diff
@@ -0,0 +1,4 @@
+[Service] 
+Environment="HTTP_PROXY={{ splunk_service_proxy }}" 
+Environment="HTTPS_PROXY={{ splunk_service_proxy }}" 
```

Comment:
Do you think that there won't be a use case when these vars need to be set to different values?

Reply:
I asked myself the same question and I thought it would be very unlikely but I can use 2 variables if you think it could be useful

Reply:
we can possibly set the value of one variable using the other one but I am afraid it could lead to undesired configuration by the user.

```suggestion
Environment="HTTP_PROXY={{ splunk_service_proxy_http | default({{ splunk_service_proxy_https }}) }}" 
Environment="HTTPS_PROXY={{ splunk_service_proxy_https | default({{ splunk_service_proxy_http }}) }}" 
```

If you want this so we should change the default values of these variables to `None` and the underlying `when` to check None instead of empty string to allow user to define a proxy for only one protocol

Reply:
I think explicitly asking for both is a better approach. I like the current state of the PR